modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/branding/preference/management/v1/AppBrandingPreferenceManagementSuccessTest.java

```diff
@@ -0,0 +1,710 @@
+
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_CREATED)
+                .header(HttpHeaders.LOCATION, notNullValue());
+    }
+
+    private void addOrgBrandingPreference(String orgId, String orgBrandingRequestBodyFile) throws Exception {
+
+        RestAssured.basePath = buildTenantedBasePathForOrg(tenant);
+        String body = readResource(orgBrandingRequestBodyFile)
+                .replace(ORG_NAME_PLACEHOLDER, orgId);
+        Response response = getResponseOfPostWithOAuth2(BRANDING_PREFERENCE_API_BASE_PATH, body,
+                orgMgtRestClient.switchM2MToken(orgId));
+
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_CREATED)
+                .header(HttpHeaders.LOCATION, notNullValue());
+    }
+
+    private void deleteRootOrgBrandingPreference() throws Exception {
+
+        RestAssured.basePath = basePath;
+        Response response = getResponseOfDelete(BRANDING_PREFERENCE_API_BASE_PATH + String.format
+                (PREFERENCE_COMPONENT_WITH_QUERY_PARAM, ORGANIZATION_TYPE, tenant, DEFAULT_LOCALE));
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_NO_CONTENT);
+        RestAssured.basePath = StringUtils.EMPTY;
+    }
+
+    private void deleteOrgBrandingPreference(String orgId) throws Exception {
+
+        RestAssured.basePath = buildTenantedBasePathForOrg(tenant);
+        Response response = getResponseOfDeleteWithOAuth2(BRANDING_PREFERENCE_API_BASE_PATH + String.format
+                        (PREFERENCE_COMPONENT_WITH_QUERY_PARAM, ORGANIZATION_TYPE, orgId, DEFAULT_LOCALE),
+                orgMgtRestClient.switchM2MToken(orgId));
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_NO_CONTENT);
+    }
+
+    private Response getResolvedAppBrandingInOrg(String orgId, String appId)
+            throws Exception {
+
+        RestAssured.basePath = PATH_SEPARATOR + ORGANIZATION_PATH + orgId + PATH_SEPARATOR + API_SERVER_BASE_PATH;
+        return getResponseOfGetWithOAuth2(BRANDING_PREFERENCE_RESOLVE_PATH +
+                String.format(PREFERENCE_COMPONENT_WITH_QUERY_PARAM, APPLICATION_TYPE, appId,
+                        DEFAULT_LOCALE), orgMgtRestClient.switchM2MToken(orgId));
+    }
+
+    private String buildTenantedBasePathForOrg(String tenantDomain) {
+
+        if (tenantDomain.equals(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME)) {
+            return PATH_SEPARATOR + ORGANIZATION_PATH + API_SERVER_BASE_PATH;
+        } else {
+            return PATH_SEPARATOR + TENANT_PATH + tenantDomain + PATH_SEPARATOR + ORGANIZATION_PATH +
+                    API_SERVER_BASE_PATH;
+        }
```

Comment:
```suggestion
        }
        return PATH_SEPARATOR + TENANT_PATH + tenantDomain + PATH_SEPARATOR + ORGANIZATION_PATH +
                    API_SERVER_BASE_PATH;
```

Reply:
Resolved with https://github.com/wso2/product-is/pull/20772/commits/1258e75f5c9e9525d851fd5e4143513bfc72946c